HERACLES-14 / HERACLES-16: Triggered Assessment - Physician Interpretation (new form)

marta- opened the pull request:
https://phenotips.atlassian.net/browse/HERACLES-14

marta- committed:
HERACLES-14 / HERACLES-16: Triggered Assessment - Physician Interpretation (new form)

marta- committed:
HERACLES-14 / HERACLES-16: Triggered Assessment - Physician Interpretation (new form)
* better section structure on first page
* fixes to option letter casing
* final page - updated options and conditions for info boxes

marta- committed:
HERACLES-14 / HERACLES-16: Triggered Assessment - Physician Interpretation (new form)
* minor option label fixes
* condition fix on the last page

marta- committed:
HERACLES-14 / HERACLES-16: Triggered Assessment - Physician Interpretation (new form)
* Minor typo
* Fixed form name capitalization

marta- committed:
HERACLES-14 / HERACLES-16: Triggered Assessment - Physician Interpretation (new form)
* Fixed form name capitalization (again)
* Removed trailing whitespaces

marta- committed:
HERACLES-14 / HERACLES-16: Triggered Assessment - Physician Interpretation (new form)
Last page - reorganized info box sections, updated conditions

sashaandjic reviewed (approved):
tested

Hajany reviewed (commented):
Condition not clearing once de-selected
[screen-capture (8).webm](https://github.com/data-team-uhn/cards/assets/96542282/9edab0a3-60ac-4718-a4f6-4181d70ecd58)

marta- committed:
HERACLES-14 / HERACLES-16: Triggered Assessment - Physician Interpretation (new form)
Last page - updated infobox conditions to avoid ghost answer value bug

marta- committed:
HERACLES-14 / HERACLES-16: Triggered Assessment - Physician Interpretation (new form)
Fixed console warning caused by unsupported display mode ("input") for referemce question

sashaandjic reviewed (approved):
tested

Hajany reviewed (approved):
tested

marta- committed:
HERACLES-14 / HERACLES-16: Triggered Assessment - Physician Interpretation (new form)
Do not restrict node id length to 40 characters

sashaandjic reviewed (approved):
tested

marta- committed:
HERACLES-14 / HERACLES-16: Triggered Assessment - Physician Interpretation (new form)
Renamed to "Triggered Assessments"

sashaandjic reviewed (approved):
tested

![image](https://github.com/data-team-uhn/cards/assets/23243244/b4e9877d-4a3f-41cc-93fb-9de0ff460069)

![image](https://github.com/data-team-uhn/cards/assets/23243244/d03942df-a756-4fd1-b404-251150ae4ab3)

sdumitriu reviewed (commented):
There's an Info box at the end of the form which only shows up in edit mode. Should it be a computed, readonly question instead so it can show up in view mode?
